<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn</class>
 <widget class="QWidget" name="SignIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>890</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>890</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>890</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="titleFrame" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titleSignUp1">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleSignUp2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It's quick and easy.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="mainFrame" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QWidget" name="firstNameFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="firstNameLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>First Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firstNameLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="lastNameFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="lastNameLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Last Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lastNameLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="eMailFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="eMailLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eMailLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="userNameFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="userNameLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>User Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userNameLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="passwordFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="passwordLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="addressFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="addressLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addressLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="phoneNumberFrame" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="phoneNumberLabel">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Phone Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phoneNumberLineEdit">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QPushButton" name="signUpBtn">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="clickableLabel" name="backToLogInBtn">
     <property name="text">
      <string>Back to Log In</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>clickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
